go.sum

```diff
@@ -9,7 +9,6 @@ cloud.google.com/go v0.43.0/go.mod h1:BOSR3VbTLkk6FDC/TcffxP4NF/FFBGA5ku+jvKOP7p
 cloud.google.com/go v0.44.1/go.mod h1:iSa0KzasP4Uvy3f1mN/7PiObzGgflwredwwASm/v6AU=
 cloud.google.com/go v0.44.2/go.mod h1:60680Gw3Yr4ikxnPRS/oxxkBccT6SA1yMk63TGekxKY=
 cloud.google.com/go v0.45.1/go.mod h1:RpBamKRgapWJb87xiFSdk4g1CME7QZg3uwTez+TSTjc=
-cloud.google.com/go v0.46.3 h1:AVXDdKsrtX33oR9fbCMu/+c1o8Ofjq6Ku/MInaLVg5Y=
```

Comment:
go mod tidy?

Reply:
ran go mod tidy, looks like it did something/remove some lines, I don't have too much knowledge on the tool overall but less lines is better? 

https://github.com/smartcontractkit/chainlink/pull/4102/commits/fb6ad4575bdb891ec905ede24c692b82e82d79e2